test/suites/common-tanssi/fees/test_fee_burning.ts

```diff
@@ -48,7 +48,9 @@ describeSuite({
                 const baseWeightToFee = (
                     await polkadotJs.call.transactionPaymentApi.queryWeightToFee({
                         refTime: baseWeight,
-                        proofSize: feeInfo.weight.proofSize.toBigInt(),
+                        // Base weight represents the fixed computational cost, independent of proof size.
+                        // Setting proofSize to 0 ensures we only calculate the ref_time component of the base fee.
+                        proofSize: 0n,
```

Comment:
we should use proofSize: 0 here, because the original formula for base fee doesn't include proof size: https://github.com/moondance-labs/polkadot-sdk/blob/ae4b68b36d6da1dd0d6a78643b634296b3fa8039/substrate/frame/support/src/weights/extrinsic_weights.rs#L55